tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DynamicConfigurationTests.cs

```diff
@@ -158,6 +159,14 @@ private async Task UpdateAndValidateConfig(MockTracerAgent agent, LogEntryWatche
 
             var request = await agent.SetupRcmAndWait(Output, new[] { ((object)new { lib_config = config }, DynamicConfigurationManager.ProductName, fileId) });
 
+            // Validate capabilities
+            var capabilities = new BitArray(request.Client.Capabilities);
+
+            capabilities[12].Should().BeTrue(); // APM_TRACING_SAMPLE_RATE
+            capabilities[13].Should().BeTrue(); // APM_TRACING_LOGS_INJECTION
+            capabilities[14].Should().BeTrue(); // APM_TRACING_HTTP_HEADER_TAGS
+            capabilities[15].Should().BeTrue(); // APM_TRACING_CUSTOM_TAGS
+
```

Comment:
It's a tiny nit, but if it were me, I would also add a unit test that reads all the static fields from `RcmCapabilitiesIndices` using reflection, and asserts that they're all different, and that all values from 1-<max> are represented

Reply:
Makes sense. Added in https://github.com/DataDog/dd-trace-dotnet/pull/4928/commits/9bf13e95679ba30ca24e52f809b0520c877c0a59